### PR TITLE
Add more info allow list

### PR DESCRIPTION
**Pull request by tbruno**
Explain each address that needs to be added to the allow list.
Change terminology from whitelist to allow list.

### CODE REVIEW

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 1]
```diff
@@ -59,25 +59,25 @@ For more information about which permissions to use, see xref:api-manager::org-c
...
-|*EU*|**.dkr.ecr.eu-central-1.amazonaws.com* |  443
-|*EU*|*exchange-files.eu1.anypoint.mulesoft.com* |  443
-|*EU*|*exchange2-asset-manager-kprod-eu.s3.eu-central-1.amazonaws.com* |  443
-|*ALL*|*get.helm.sh* | 443
-|*ALL*|*storage.googleapis.com* | 443
+|Plane |Host |Port| Explanation
+|*US*|*anypoint.mulesoft.com* | 443| Required to connect with the control plane
+|*US*|*analytics-ingest.anypoint.mulesoft.com* |  443| Required to send analytics to the control plane
+|*US*|**.dkr.ecr.us-east-1.amazonaws.com* |  443| Required to download the docker images that make up the Anypoint Service Mesh product
+|*US*|*exchange-files.anypoint.mulesoft.com* |  443| Required to download the docker images that make up the Anypoint Service Mesh product
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: I don't think this is correct

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 2]
```diff
@@ -59,25 +59,25 @@ For more information about which permissions to use, see xref:api-manager::org-c
...
+|*US*|*exchange2-asset-manager-kprod.s3.amazonaws.com* |  443| Required to download policies
+|*EU*|*eu1.anypoint.mulesoft.com* | 443| Anypoint
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: `Required to connect with the control plane`?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (dosten) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: `Required to download policies`?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 3]
```diff
@@ -59,25 +59,25 @@ For more information about which permissions to use, see xref:api-manager::org-c
...
+|*EU*|*analytics-ingest.eu1.anypoint.mulesoft.com* |  443| Required to send analytics to the control plane
+|*EU*|**.dkr.ecr.eu-central-1.amazonaws.com* |  443| Required to download the docker images that make up the Anypoint Service Mesh product
+|*EU*|*exchange-files.eu1.anypoint.mulesoft.com* |  443| Required to download policies
+|*EU*|*exchange2-asset-manager-kprod-eu.s3.eu-central-1.amazonaws.com* |  443| Required to download policies
+|*ALL*|*get.helm.sh* | 443| Required to retrieve and install Helm on the Kubernetes clusters.
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: This seems wrong, we don't install Helm into the cluster, only into the users's machine

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 4]
```diff
@@ -59,25 +59,25 @@ For more information about which permissions to use, see xref:api-manager::org-c
...
+|*ALL*|*storage.googleapis.com* | 443| Required to retrieve and install kubectl on the Kubernetes clusters.
```
Comment (dosten) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Same here

**Review by dosten — CHANGES_REQUESTED** [review]

**Comment by tbruno** [comment]
Thanks for the feedback, feedback applied @dosten

**Review by dhirsch1 — APPROVED** [review]

**Review by dosten — APPROVED** [review]

**Pull request merged**